src/prx/main.py

```diff
@@ -447,9 +446,17 @@ def process(observation_file_path: Path, output_format="jsonseq"):
     aux_files = nav_file_discovery.discover_or_download_auxiliary_files(
         rinex_3_obs_file
     )
+    metadata = build_metadata(
+        {"obs_file": rinex_3_obs_file, "nav_file": aux_files["broadcast_ephemerides"]}
+    )
+    records = build_records(
+        rinex_3_obs_file,
+        aux_files["broadcast_ephemerides"],
+        metadata["approximate_receiver_ecef_position_m"],
```

Comment:
I see that this `approximate_receiver_ecef_position_m` is useful in our case (processing observations from fixed station), but I still think that a more detailed receiver position is needed, per epoch.

This is a separate issue, that I detail in #60.

So let us keep this as it is for this PR, but begin to think about more complex cases;

Reply:
I just checked, the RNX header field `APPROX POSITION XYZ` is an optional field...

Reply:
Yes, thanks for opening https://github.com/jtec/prx/issues/60, it will be great to make that more robust.